ios/Classes/codec/AblyPlatformConstants.h

```diff
@@ -164,3 +170,21 @@ extern NSString *const TxMessage_data;
 extern NSString *const TxMessage_name;
 extern NSString *const TxMessage_extras;
 @end
+
+@interface TxPaginatedResult : NSObject
```

Comment:
These `@interface` declarations and the corresponding `@implementation` blocks do nothing. I would prefer that we remove them as the inference is that they add value, but they don't.